src/App.tsx

```diff
@@ -1,16 +1,22 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState([...moviesFromServer]);
+
+  const handleMovie = (movie: Movie) => setMovies([...movies, movie]);
```

Comment:
use functional setState
```suggestion
  const handleMovie = (movie: Movie) => setMovies((...) => ...);
```